development/build/scripts.js

```diff
@@ -50,6 +51,42 @@ const {
   createRemoveFencedCodeTransform,
 } = require('./transforms/remove-fenced-code');
 
+// map dist files to bag of needed native APIs against LM scuttling
+const scuttlingBagConfig = {
```

Comment:
"bag" is very colloquial and generally not used in code/comments. `scuttlingConfig` should suffice

Reply:
0858e0e29